docs/conf.py

```diff
@@ -0,0 +1,75 @@
+#
+# Configuration file for the Sphinx documentation builder.
+#
+
+# get the version from the inventory file
+from pathlib import Path
+
+import yaml
+
+import disk_objectstore
+
+# -- Project information -----------------------------------------------------
+
+project = "disk-objectstore"
+copyright = "2020-2022, The AiiDA team"
+author = "The AiiDA team."
+
+# The short X.Y version.
+version = ".".join(disk_objectstore.__version__.split(".")[:2])
+# The full version, including alpha/beta/rc tags.
+release = disk_objectstore.__version__
+
+# -- General configuration ---------------------------------------------------
+
+needs_sphinx = "2.0"
+extensions = [
+    "myst_parser",
+    "sphinx_panels",
+    "ablog",
```

Comment:
don't think ablog is necessary here (guess you copy/pasted from quantum-mobile docs?)